lib/components/molecules/AccordionBasic/AccordionBasic.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+/**
+ *
+ * AccordionBasic
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import type { Node } from 'react';
+import styles from './AccordionBasic.style';
+
+type Props = {
+  className?: string,
+  title?: string,
+  value?: string,
+  isOpen?: boolean,
+};
+
+const AccordionBasic = ({ className, title, value, isOpen }: Props): Node => (
+  	<li className={className}>
```

Comment:
This is absolutely wrong, you can't expect an component to be wrapped into another component when they use this.